src/psyclone/tests/psyad/adjoint_visitor_test.py

```diff
@@ -498,19 +498,19 @@ def test_loop_node_bounds_error(fortran_reader):
     with pytest.raises(VisitorError) as info:
         _ = adj_visitor(tl_psyir)
     assert ("The lower bound of a loop should not contain active variables, "
-            "but found 'lo'" in str(info.value))
+            "but found lo in 'lo'" in str(info.value))
```

Comment:
I think this would be nicer with the variable name in quotes too.

Reply:
Updated.